modules/hivemq-edge-module-plc4x/src/main/java/com/hivemq/edge/adapters/plc4x/types/ab/ABProtocolAdapterInformation.java

```diff
@@ -56,24 +55,39 @@ protected ABProtocolAdapterInformation() {
     }
 
     @Override
-    public String getVersion() {
-        return super.getVersion() + " (BETA)";
+    public @NotNull String getUrl() {
+        return "TODO";
```

Comment:
TODO

Reply:
oh thats from rebasing. That shouldnt be here